portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,62 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+    <head>
+        <meta charset="UTF-8">
+        <title>Tony Shen - Home</title>
+        <link rel="stylesheet" href="style.css">
+        <!--Google Font API-->
+        <link rel="stylesheet"
+          href="https://fonts.googleapis.com/css?family=Roboto&effect=anaglyph">
+        <script src="script.js"></script>
+    </head>
+    <body style="page-container">
+        <div class="axis-0-grid-container">
+            <div class="header-container">
+
+                <div class="logo-section">
+                    <img class="logo" src="images/profile.jpg"/>
+                </div>
+
+                <div class="title-section"><a class="font-effect-anaglyph title" href="index.html">Tony Shen</a></div>
+
+                <div class="navigator-section">  
+                    <a class="font-effect-anaglyph header-links" href="projects.html">Projects</a>
+                    <a class="font-effect-anaglyph header-links" href="https://drive.google.com/file/d/1lSsqU0wnAMAjw4Yj3lWPOef8z43ro9un/view?usp=sharing">CV</a>
+                    <a class="font-effect-anaglyph header-links" href="blog.html">Blog</a>
+                    <a class="font-effect-anaglyph header-links" href="gallery.html">Images</a>
+                </div>
+
+            </div>
+            <div class="aboutme-container">
+                <div class="subtitle">Mission: </div>
+                <div class="text">Create and explore technologies to improve the future of humanity.</div>
+            </div>
+
+            <div class="grid-container">      
```

Comment:
wait, why do I see it again... I'm a bit confused. In such case it would be good to have a description of your change that outlines all the pages added and their expected content.

Reply:
Could be remnants of refactoring. I moved all the html files to a folder, but then realized index.html has to stay in the project root for "maven" to run. 

I will try to avoid pushing next time and add a comment before every HTML file for clarity.